test/integration-test/PassStageMerger/Test05_after_ref.json

```diff
@@ -13,7 +13,7 @@
           },
           "Stage_1": {
             "Do_0": {
-              "Accesses_0": "W:field_a2:<0,0,0,0,0,0> R:field_a1:<0,1,0,0,0,0>",
+              "Accesses_0": "W:field_a2:<0,0,0,0,0,0> R:field_a1:<1,1,0,0,0,0>",
```

Comment:
again, the reference files should be recompiled once the stencil-function access issue is resolved